cli/src/test/resources/ide-projects/tomcat/repository/java/java/default/readme

```diff
@@ -0,0 +1 @@
+Readme file
```

Comment:
The software installation should contain something more reasonable that you also assert in your test to see that the proper thing was installed. Here are multiple `readme` files that contain the wise words `Readme file` so in case the wrong `readme` file was copied, your test could not even detect this.
Due to my concern about the more complex case to be covered by the test, I would recommend to have two `java` versions like some 11 and some 21 version and have one configured as `JAVA_VERSION` that is incompatible with the `TOMCAT_VERSION` that would require the other and then test that starting tomcat did use the second one.

Reply:
I have removed the unnecessary readme files and for the folders I want to have included in github, I have added a couple of .gitkeep files